lib/razorpay/validation_config.rb

```diff
@@ -0,0 +1,19 @@
+module Razorpay
+  ValidationType = {
```

Comment:
Where is this getting used?

Reply:
https://github.com/razorpay/razorpay-ruby/blob/be0455b3719eab23913e6fca28512c869782199b/lib/razorpay/payload_validator.rb#L28